src/AbstractAlgebra.jl

```diff
@@ -560,8 +560,9 @@ function direct_sum(m::Module{T}...) where T <: RingElement
    Generic.DirectSum(m...)
 end
 
-function ModuleHomomorphism(M1::AbstractAlgebra.Module, M2::AbstractAlgebra.Module, m::MatElem)
-   Generic.ModuleHomomorphism(M1, M2, m)
+#why? Why not just export the Generic function?
```

Comment:
Generic contains functions that may be overridden by external modules, e.g. Nemo. It is useful to be able to call the Generic version directly for comparison.

It is better to open tickets rather than leave questions in the source code for other developers, which probably won't get read or answered.

Can we now please remove this comment.